src/olympia/addons/templates/addons/persona_preview.html

```diff
@@ -25,7 +25,7 @@
             {# L10n: {0} is the number of users. #}
             {{ ngettext("<strong>{0}</strong> user",
                         "<strong>{0}</strong> users",
-                        addon.persona.popularity)|f(addon.persona.popularity|numberfmt)|safe }}
+                        addon.persona.popularity)|fe(addon.persona.popularity|numberfmt)|safe }}
```

Comment:
Are you sure about the `|safe` ? It's really hard to read, but since the `|safe` call it ouside the parenthesis, I assumed it applied to the whole `ngettext()` call.